app/views/questionnaire.py

```diff
@@ -457,6 +463,26 @@ def _get_front_end_navigation(answer_store, current_location, metadata):
         return None
 
 
+def get_page_title_for_block(schema_json, current_location):
```

Comment:
To me it feels like this should be passing a block and not current_location. Obviously we would then need a helper like get_group_for_block if we don't have something like that already.